test/jdk/javax/sound/SoundClip/SoundClipTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.File;
+import javax.sound.SoundClip;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.Mixer;
+
+/*
+ * @test
+ * @bug 8356049
+ * @key sound headful
+ * @summary basic testing of javax.sound.SoundClip
+ * @run main/othervm SoundClipTest javasound.wav
+ */
+
+public class SoundClipTest {
+
+    public static void main(String[] args) throws Exception {
+
+        if (!isSoundcardInstalled()) {
```

Comment:
Why we cannot check that the playing sound is a no-op in this case, as specified?

Reply:
But it is part of the new spec:
>     * if the implementation does not find a suitable output device for the data,
>     * playing the clip will be a no-op.

It is better to cover it if we have a chance than just exiting.